src/classes/PanelData.js

```diff
@@ -348,7 +348,7 @@ class PanelData {
 			.set('pageHost', pageHost)
 			.set('performanceData', tab && tabInfo.getTabInfo(tab_id, 'pageTiming'))
 			.set('sitePolicy', tab && policy.getSitePolicy(tab_url) || false)
-			.set('siteNotScanned', tab && !foundBugs.getApps(tab_id) || false)
+			.set('siteNotScanned', tab && !trackerList.length || false)
 			.set('tab_id', tab_id)
```

Comment:
@zarembsky `foundBugs.getApps()` returns 3 states, false for page not scanned, `[]` for page scanned but found no trackers, and `[tracker, tracker, ...]` for page scanned and trackers were found. So we should leave this as it was.